### ISSUE TITLE
[pkgconf:x64-windows] meson.build error: Value "plain" .. for combo option "Optimization level" is not one of the choices

### ISSUE THREAD

**Issue by meghadri** [microsoft/vcpkg#37393]
Content of meson-log.txt

```
Build started at 2024-03-12T21:36:09.828642
Main binary: V:\downloads\tools\python\python-3.11.8-x64\python.exe
Build Options: -Dtests=disabled -Dbuildtype=plain -Doptimization=plain -Ddebug=true '-Dcmake_prefix_path=['"'"'F:/code/cmake-build-debug/vcpkg_installed/x64-windows/debug'"'"','"'"'F:/code/cmake-build-debug/vcpkg_installed/x64-windows'"'"','"'"'F:/code/cmake-build-debug/vcpkg_installed/x64-windows/share'"'"']' -Dprefix=V:/packages/pkgconf_x64-windows/debug -Dincludedir=../include -Dlibdir=lib -Dbackend=ninja -Dwrap_mode=nodownload '--native-file V://pkgconf/meson-x64-windows-dbg.log'
Python system: Windows
The Meson build system
Version: 0.62.1
Source dir: F:\tools\vcpkg\pkgconf\src\conf-2.1.0-7b1cf1fe7f.clean
Build dir: F:\tools\vcpkg\pkgconf\x64-windows-dbg
Build type: native build

F:\tools\vcpkg\pkgconf\src\conf-2.1.0-7b1cf1fe7f.clean\meson.build:1:0: ERROR: Value "plain" (of type "string") for combo option "Optimization level" is not one of the choices. Possible choices are (as string): "0", "g", "1", "2", "3", "s".
meson

```

1. This issue occurred unexpectedly, viz. no changes to the project config or the vcpkg manifest file. 
2. vcpkg install directory is F:\tools\vcpkg. This has been mapped to V: to workaround failures building Qt, parts of which seem to hit a long path issue. 
 
Have tried git pull and vcpkg update without any success. 
All help with this appreciated.

**Comment by Neumann-A** [comment]
Should be fixed with #37396

**Comment by meghadri** [comment]
Thanks. Subsequently, there was an error in harfbuzz which failed in gen-harfbuzzcc.py at

```
Traceback (most recent call last):
  File "F:\tools\vcpkg\harfbuzz\src\8.3.0-222d759c60.clean\src\gen-harfbuzzcc.py", line 17, in <module>
    f.write ("".join ('#include "{}"\n'.format (os.path.relpath (os.path.abspath (x), CURRENT_SOURCE_DIR)) for x in sources if x.endswith (".cc")).encode ())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\tools\vcpkg\harfbuzz\src\8.3.0-222d759c60.clean\src\gen-harfbuzzcc.py", line 17, in <genexpr>
    f.write ("".join ('#include "{}"\n'.format (os.path.relpath (os.path.abspath (x), CURRENT_SOURCE_DIR)) for x in sources if x.endswith (".cc")).encode ())
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```
This was fixed by setting VCPKG_ROOT to the actual location of the VCPKG installation. Remains to be seen how Qt reacts with these changes. 

Thanks for the quick response.
